app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
Yep, makes sense, though we would want to show a reason that the button is disabled (i.e. "Assigned rates can not be removed.") so the method would have to provide that.